src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 public class FileWork {
+    private static final String LETTER_WORDS_SHOULD_STARTS_WITH = "w";
+    private static final String DELIMITER = " ";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        List<String> result = new ArrayList<>();
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+
+                for (String word : value.split(DELIMITER)) {
+                    if (word.toLowerCase().startsWith(LETTER_WORDS_SHOULD_STARTS_WITH)) {
+                        result.add(word.toLowerCase().replaceAll("[!?,.]", ""));
+                    }
+                }
+                value = bufferedReader.readLine();
```

Comment:
you can read new line in loop condition, than it will read a line with every iteration